crypto/test/aes_calc.c

```diff
@@ -44,117 +44,113 @@
  */
 
 /*
-  
+
  Example usage (with first NIST FIPS 197 test case):
- 
-[sh]$ test/aes_calc 000102030405060708090a0b0c0d0e0f 00112233445566778899aabbccddeeff -v
+
+[sh]$ test/aes_calc 000102030405060708090a0b0c0d0e0f
```

Comment:
Maybe insert a \\ character in the comments and indent the second line to make it more readable?

Reply:
I agree, when the tool breaks comments it usually makes a mess.